src/components/pages/GenerationPage/GenerationPage.jsx

```diff
@@ -186,7 +186,6 @@ function GenerationPage() {
             </Button>
             <Button
               onClick={() => generateRecordsSpreadsheet()}
-              disabled={!!data?.generate_summary_errors?.batch}
               title={data?.generate_summary_errors?.batch?.join(' ') ?? ''}
```

Comment:
This should be removed as well. Otherwise, it will show errors for the account summary generation while hovering over the button